src/utilities/util_verify.c

```diff
@@ -31,20 +31,43 @@ usage(void)
     return (1);
 }
 
+/*
+ * verify_one --
+ *     Verify the file specified by the URI.
+ */
+static int
+verify_one(WT_SESSION *session, char *config, char *uri)
+{
+    WT_DECL_RET;
+
+    if ((ret = session->verify(session, uri, config)) != 0)
+        ret = util_err(session, ret, "session.verify: %s", uri);
+    else {
```

Comment:
We can probably improve this by writing `else if(verbose)` directly

Reply:
Good suggestion, thanks!